src/Rfc6455/MessageProcessor.php

```diff
@@ -69,7 +68,10 @@ public function onData(string $data, ConnectionInterface $socket, Message $messa
                 } else {
                     yield $message;
                 }
-
+            } catch (TooBigControlFrameException $e) {
```

Comment:
There's a problem here:

*The `TooBigControlFrameException` inherits from `TooBigFrameException` which looks good but isn't because the `TooBigFrameException` actually inherits from `LimitationException`, which is ok but `TooBigControlFrameException` should inherits from `ProtocoleErrorException` or something like that, so we can catch every protocol error easily.*

This is a logical problem because **for now**, a `TooBigControlFrameException` actually **is** a `LimitationException`.